Cargo.toml

```diff
@@ -25,6 +26,13 @@ exclude = [".github/"]
 [workspace.dependencies]
 miden-air = { version = "0.9", default-features = false }
 miden-lib = { version = "0.2"}
+miden-node-block-producer = { path = "crates/block-producer", version = "0.2" }
```

Comment:
I think, it's not needed to specify version here.

Reply:
removed.